app/src/main/res/layout/activity_maps.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"

```

Comment:
I am wondering is it better to use ConstraintLayout compared to RelativeLayout? [](https://android-developers.googleblog.com/2017/08/understanding-performance-benefits-of.html)